testapi.pm

```diff
@@ -390,8 +390,9 @@ sub upload_logs {
     my ($file) = @_;
 
     bmwqemu::log_call('upload_logs', file => $file);
-    my $cmd      = "curl --form upload=\@$file ";
     my $basename = basename($file);
+    my $upname   = ref($autotest::current_test) . '-' . $basename;
```

Comment:
To me it just a too long filename to me, and I can't see the use case that the log will be overwrited  from anything else, do we have duplicate module name?

I'm fine with the long name if it's preferred. What other people think here? :)